pages/vi/profiles/Kid243.md

```diff
@@ -0,0 +1,6 @@
+# Shalom brethren
+My name is __Dijimba Kabuyi Ilunga___ . I go by _Joss_ most of the time. I reside currently in Arlington, Texas (central standard time). My operating system is Windows 10.
```

Comment:
Could you please remove extra **underscore** character at the end of your name.

Reply:
Alright getting to it